plugins/json_logs.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.0.1
+title: JSON
+description: Log parser for JSON
+parameters:
+  - name: log_paths
+    type: "[]string"
+  - name: exclude_log_paths
+    type: "[]string"
+    default: []
+  - name: encoding
+    type: string
+    supported:
+      - nop
+      - utf-8
+      - utf-16le
+      - utf-16be
+    default: utf-8
+  - name: log_type
+    type: string
+    default: json
```

Comment:
@BinaryFissionGames would this ever make sense to be anything different than json? should this just be hard coded in the template?

Reply:
gotcha